src/containers/Interfaces/AlterForm.js

```diff
@@ -126,6 +127,7 @@ class AlterForm extends Component {
               subject={stage.subject}
               node={node}
               index={index}
```

Comment:
Should we change these to `edgeIndex`/`nodeIndex` now we have multiple indexes?

Reply:
Good point!